activerecord/test/cases/scoping/default_scoping_test.rb

```diff
@@ -193,7 +193,7 @@ def test_unscope_with_limit_in_query
 
   def test_order_to_unscope_reordering
     scope = DeveloperOrderedBySalary.order("salary DESC, name ASC").reverse_order.unscope(:order)
-    assert_not /order/i.match?(scope.to_sql)
+    assert_not (/order/i.match?(scope.to_sql))
```

Comment:
Can you use `assert_no_match`?

Reply:
Thanks for the suggestion as always. Yes, `assert_no_match` works fine.